lib/js/components/Buttons/Button/Button.vue

```diff
@@ -21,7 +21,6 @@
 			'-elevation-x-small': elevation === ELEVATIONS.X_SMALL,
 			'-elevation-small': elevation === ELEVATIONS.SMALL,
 		}"
-		@click="$emit('click')"
```

Comment:
Will we have to update all button uses in platform (`@click.native`)? Is it necessary because of https://github.com/vuejs/core/issues/813? We could always use `emits: ["click"]` but maybe it's better to go straight for the best solution.

Reply:
after upgrade, `.native` modifier is ignored and all events are handled by `@click`. By removing this line, we don't emit duplicated event in this component, so migration in platform should be limited to `@click.native => @click`